packages/extension-utils/src/browserAdaptor.ts

```diff
@@ -24,23 +24,41 @@
 
  */
 
-import type { FC } from 'react'
-import React from 'react'
-import { Provider } from 'react-redux'
-import { ExtensionProvider } from '@looker/extension-sdk-react'
-import { ComponentsProvider } from '@looker/components'
-import { hot } from 'react-hot-loader/root'
-import { Hackathon } from './Hackathon'
-import store from './data/store'
-
-export const App: FC = hot(() => {
-  return (
-    <Provider store={store}>
-      <ExtensionProvider>
-        <ComponentsProvider>
-          <Hackathon />
-        </ComponentsProvider>
-      </ExtensionProvider>
-    </Provider>
-  )
-})
+import type { IExtensionAdaptor, ThemeOverrides } from './adaptorUtils'
+import { getThemeOverrides, hostedInternally } from './adaptorUtils'
+
+/**
+ * An adaptor class for interacting with browser APIs when not running in an extension
+ */
+export class BrowserAdaptor implements IExtensionAdaptor {
```

Comment:
Ignore that git thinks this was a migration from `App.tsx`. That's completely and utterly bogus.